packages/vanilla/README.md

```diff
@@ -1,26 +1,31 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
+
 ### Complex Forms in the blink of an eye
 
-JSONForms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+JSON Forms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+
+## Vanilla Renderers Package
+
```

Comment:
Also mention and link the React seed and the React documentation on JSON forms website.